invenio_accounts/templates/invenio_accounts/forgot_password.html

```diff
@@ -34,7 +34,7 @@ <h3 class="text-center panel-free-title">{{_('Reset Password')}}</h3>
     {# contents of this if-block can be always shown after mattupstate/flask-security#291 is released. #}
     {%- if current_user.is_anonymous %}
     <div class="panel-footer text-center">
-        <h4 class="text-muted"><a href="{{url_for('security.login')}}">{{_('Log In')}}</a>{% if security.registerable %} {{_('or')}} <a href="{{url_for('security.register')}}">{{_('Sign Up')}}</a>{% endif %}</h4>
+        <h4 class="text-muted"><a href="{{url_for('security.login')}}">{{_('Sign In')}}</a>{% if security.registerable %} {{_('or')}} <a href="{{url_for('security.register')}}">{{_('Sign Up')}}</a>{% endif %}</h4>
```

Comment:
Nice catch 🔥

Reply:
Minor: We terms "Log in" and "Sign up" are correct. We should not use "Sign in".